src/index.html

```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonimous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img class="img" src="images/logo.png" alt="logo">
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li is-active">Apple</li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active">
              Apple
            </a>
          </li>
```
Елементи меню мають бути посиланнями, щоб можна було по ним клікати